tdx-skills/parent-segment/SKILL.md

```diff
@@ -0,0 +1,188 @@
+---
+name: parent-segment
+description: Manage CDP parent segments with tdx CLI using YAML-based configuration for master tables, attributes, and behaviors. Use when setting up parent segments, validating configurations, or running parent segment workflows.
+---
+
+# tdx Parent-Segment - CDP Parent Segment Management
+
+Manage CDP parent segments using `tdx parent-segment` (alias: `tdx ps`) with YAML-based configuration.
+
+## What is a Parent Segment?
+
+A parent segment defines the master customer table enriched with:
+- **Master table**: Core customer/user records
+- **Attributes**: Customer properties (user profiles, account info)
+- **Behaviors**: Customer actions (purchases, page views, events)
+
+Parent workflows create output database `cdp_audience_xxx` with enriched `customers` table for child segments.
+
+## Core Commands
+
+```bash
+# List parent segments
+tdx ps list
+tdx ps list "Customer*"  # Filter with pattern
+
+# Pull configuration to YAML (auto-sets context)
+tdx ps pull "Customer 360"
+# Creates: parent_segments/customer-360.yml
+
+# Push configuration (create/update)
+tdx ps push "Customer 360"
```

Comment:
This command sets parent_segment context to "Customer 360" (parent_segemnts/customer-360.yml file). So the subsequent commands doesn't need to specify "Customer 360"